packages/collapsible/demo/CustomCollapsible.js

```diff
@@ -66,20 +64,22 @@ export class CustomCollapsible extends LionCollapsible {
   /**
    * Trigger hide animation and wait for transition to be finished.
    * @param {Object} options - element node and its options
+   * @param {HTMLElement} options.contentNode
    * @override
    */
   async _hideAnimation({ contentNode }) {
     if (this._contentHeight === '0px') {
       return;
     }
-    ['opacity', 'padding', 'max-height'].map(prop => contentNode.style.setProperty(prop, 0));
+    ['opacity', 'padding', 'max-height'].map(prop => contentNode.style.setProperty(prop, `${0}`));
```

Comment:
```suggestion
    ['opacity', 'padding', 'max-height'].map(prop => contentNode.style.setProperty(prop, '0'));
```

should be good enough right 😅

Reply:
Hehehe yeah, my brain was on autopilot for these type issues and just put `${}` without thinking much about it haha